mails/themes/modern_mjml/core/order_customer_comment.mjml.twig

```diff
@@ -10,7 +10,7 @@
       <!-- TITLE BEGINING -->
     </mj-raw>
     <mj-text padding-top="0" padding-bottom="20px" font-weight="600" font-size="20px">
-      {{ 'Hi,'|trans({}, 'Emails.Body', locale) }}
+      {{ 'Hi {firstname} {lastname},'|trans({}, 'Emails.Body', locale) }}
```

Comment:
As @okom3pom said, this email [is for the shop, not the customer](https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/front/OrderDetailController.php#L93).
```suggestion
      {{ 'Hi,'|trans({}, 'Emails.Body', locale) }}
```